lib/WorseReflection/Bridge/TolerantParser/Diagnostics/DeprecatedUsageDiagnosticProvider.php

```diff
@@ -220,6 +220,36 @@ public function foo(Closure $foobar) {
                 Assert::assertEquals('Call to deprecated property "deprecated": This is deprecated', $diagnostics->at(0)->message());
             }
         );
+
+        yield new DiagnosticExample(
+            title: 'Deprecate with replacement',
+            source: <<<'PHP'
+                <?php
+
+                class SomeClass {
+                    /**
+                    * @deprecated Will be removed
+                    *
+                    * @see SomeClass::new
```

Comment:
This should have parenthesis:
```suggestion
                    * @see SomeClass::new()
```

`SomeClass::new` points to a property `new` instead of the method `new()`

Reply:
True, I mean *technically* this is the value of the new call so if you wanted a function you technically have to do this: `SomeClass::new(...)`. But you are right, adding braces to signify it's the function makes sense